src/stores/AccountController.ts

```diff
@@ -501,3 +503,12 @@ const validateInputLength = (values: { firstName: string; lastName: string }) =>
 
   return errors;
 };
+
+export const getReceipt = async (transactionId: string) => {
+  const { cleengSandbox } = useConfigStore.getState().getCleengData();
+  const { auth } = useAccountStore.getState();
+
+  if (!auth) throw new Error('user is not logged in');
+
+  return (await subscriptionService.fetchReceipt({ transactionId }, cleengSandbox, auth.jwt)).responseData;
```

Comment:
- Can you use the `useAccount` and `useService` helpers here?
- We should check if the subscriptionService contains the `fetchReceipt` method and throw if not defined (InPlayer integration)